servers/rendering/renderer_rd/effects_rd.cpp

```diff
@@ -59,7 +59,7 @@ RID EffectsRD::_get_uniform_set_from_image(RID p_image) {
 	u.ids.push_back(p_image);
 	uniforms.push_back(u);
 	//any thing with the same configuration (one texture in binding 0 for set 0), is good
-	RID uniform_set = RD::get_singleton()->uniform_set_create(uniforms, luminance_reduce.shader.version_get_shader(luminance_reduce.shader_version, 0), 1);
+	RID uniform_set = RD::get_singleton()->uniform_set_create(uniforms, roughness.shader.version_get_shader(roughness.shader_version, 0), 1);
 
```

Comment:
So these are the regression fixesses. As the `luminance_reduce` shaders are now unused and not initialised on the mobile renderer, when we abuse them in this way to setup uniform sets for other shaders, stuff break... 

As discussed with Reduz, I plan to split all the effects into their own classes to prevent stuff like this.

Reply:
This has now been dealt with by rewriting these shaders to fragment shaders in another PR that was already merged.